Python/Product/Analysis/Interpreter/Ast/AstPythonInterpreterFactory.cs

```diff
@@ -498,6 +498,7 @@ public sealed class TryImportModuleContext {
             CancellationToken cancellationToken
         ) {
             IPythonModule module = null;
+            bool isCelery = name == "celery";
```

Comment:
Is it case sensitive?

Reply:
It is, but this is also unused and should be deleted :)